### DISCUSSION TITLE
No "results"? #1174

### DISCUSSION THREAD

**Discussion by degenaro** [kubescape/kubescape#1175] [General]
@matthyx I'm sorry that I do not follow your answer to #1174, and that you closed the discussion. 

When I run kubescape against the Bookinfo.yaml file I get an output.json file that has section "results" comprising 38 entries. Please see attached screenshot.

![Expected "results"](https://user-images.githubusercontent.com/520029/229348137-25a24437-defa-4a82-8e7f-d14c9513ee57.png)

When I run kubescape against the example composite bitnami yaml file I posted in #1174 I do get an output.json file, but sadly the "results" section is missing. No need to involve a repo.

Please try running kubescape against the bitnami yaml file in #1174 and see that you get "summaryDetails" section but no "results" section.

**Comment by matthyx** [comment]
Sorry @degenaro I will have another look...
